.github/workflows/bot.yml

```diff
@@ -964,33 +970,50 @@ jobs:
         include:
           - sparkProfile: 'spark3.5'
             sparkArchive: 'spark-3.5.3/spark-3.5.3-bin-hadoop3.tgz'
+            scalaProfile: '-Dscala-2.12 -Dscala.binary.version=2.12'
```

Comment:
Let's add a new job for integration tests with Java 17 without affecting Spark 3.5 tests on Java 8?